buzzmobile/simulation/models/car/car_joints.urdf

```diff
@@ -342,15 +356,21 @@
     <joint name="lidar_joint" type="fixed">
         <parent link="chassis"/>
         <child link="laser"/>
-        <origin xyz="0.255 0 0.110" rpy="0 0 0"/>
+        <origin xyz="0.255 0 5" rpy="0 0 0"/>
```

Comment:
yeah, its actually the same height as the base axle and about 12 inches from the front of the chassis.